jac-uk/admin
# Display extra pupilage details

## What's included?
More logic in the application view, presenting the more involved answers for barrister qualifications.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Ensure changes made on apply side [see apply #747 for more details] show in application view.

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context
![image](https://user-images.githubusercontent.com/44227249/115870574-561f9900-a437-11eb-88b3-b1aa31fafcee.png)
![image](https://user-images.githubusercontent.com/44227249/115871171-19a06d00-a438-11eb-8612-da40fdd4bc9b.png)
![image](https://user-images.githubusercontent.com/44227249/115871212-2a50e300-a438-11eb-9b6d-90b4e4322393.png)
---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
